src/SA/reg_query/entry.c

```diff
@@ -311,7 +337,7 @@ DWORD Reg_EnumKey(const char * hostname, HKEY hivekey, DWORD Arch, const char* k
                     }
                     else
                     {
-                        internal_printf("%s%s\\%s\n", gHiveName, curitem->keypath, currentkeyname);
+                        internal_printf("%s\\%s\\%s\n", gHiveName, curitem->keypath, currentkeyname);
```

Comment:
Here we would still want the timestamp as these are not values of the queried key but rather keys within that key which may have different timestamps as shown below
![image](https://github.com/user-attachments/assets/478ac0cb-b253-4e1e-aa86-e3ca6344c6ba)

This may look something like this
![image](https://github.com/user-attachments/assets/7e7c82bd-6e4e-464b-a89a-997e9044d37a)